tests/cephadm/test_cephadm_custom_config_support.py

```diff
@@ -0,0 +1,55 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import CephadmOpsExecutionError
+from cli.ops.cephadm import generate_bootstrap_config
+from cli.utilities.containers import Container
+
+
+def check_custom_file(node, config):
+    """Check custom file updated to conatiner"""
+    for spec in config["spec"]:
+        service = spec["service_name"]
+        container_id = (
+            Container(node)
+            .ps(filter={"name": service})[0]
+            .split("\n")[1]
+            .split(" ")[0]
+            .strip()
+        )
+        mount_path = spec["custom_configs"][1]["mount_path"]
+        result = node.exec_command(
+            sudo=True, cmd=f"podman exec -ti {container_id} cat {mount_path}"
```

Comment:
add library for exec here - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py#L47

Reply:
Thanks @vamahaja 
Added